dbt_common/record.py

```diff
@@ -193,9 +199,13 @@ def pop_matching_record(self, params: Any) -> Optional[Record]:
 
         return match
 
+    def write_json(self, out_stream: TextIO):
+        d = self._to_dict()
+        json.dump(d, out_stream)
+
     def write(self) -> None:
         with open(self.current_recording_path, "w") as file:
-            json.dump(self._to_dict(), file)
+            self.write_json(file)
```

Comment:
I am assuming we are going to write to the other folder structure we discussed in a later PR?